src/conventional/ionosonde_ascii2ioda.py

```diff
@@ -18,7 +18,7 @@
 varDict = {'criticalFrequency': ['criticalFrequency', "float", 'm s-1'],
            'criticalFrequencyConfidence': ['criticalFrequencyConfidence', "float", 'fractional percent'],
            'height': ['height', "float", "m"],
-           'electronDensity': ['electronDensity', "float", 'number / m^3'],
+           'electronDensity': ['electronDensity', "float", 'number / cm^3'],
```

Comment:
```suggestion
           'electronDensity': ['electronDensity', "float", 'number cm-3'],
```

belive this is the convention we use